src/ApiCalls.ts

```diff
@@ -67,6 +67,7 @@ export async function sendFavoriteCamps(
       })
     );
 
+
     console.log("Server responses:", responses); // Log the server responses
```

Comment:
If y'all have explicitly made the decision to allow console logs into production, then this is totally fine. It's more common that console logs are not allowed past the development branch out in the wild, though.